lib/vtls/unitytls.h

```diff
@@ -1,54 +1,14 @@
 #ifndef HEADER_CURL_UNITYTLS_H
 #define HEADER_CURL_UNITYTLS_H
 
+#include "curl/curl.h"
 #include "curl_setup.h"
 
 #ifdef USE_UNITYTLS
 
-CURL_EXTERN void curl_unitytls_set_interface(unitytls_interface_struct* interface);
+CURL_EXTERN void curl_unitytls_set_interface(void* interface);
```

Comment:
Does this have to be void*? The previous version looks much nicer.

Reply:
good point, somewhat of an oversight. Fixed!